src/fetch.js

```diff
@@ -1,5 +1,6 @@
 'use strict';
 const fetch = require('node-fetch');
+const {Agent} = require('https');
```

Comment:
We need to replace "node-fetch" with https://github.com/formio/node-fetch-http-proxy since this library already handles this.  It also switches between the https agent and http agent based on the URL of the request being made.